RevenueCat/Scripts/Purchases.cs

```diff
@@ -41,56 +41,76 @@ public partial class Purchases : MonoBehaviour
     /// </summary>
     public delegate void GetPromotionalOfferFunc(PromotionalOffer promotionalOffer, Error error);
 
-    [Tooltip("Activate if you plan to call Purchases.Setup programmatically.")]
+    [Tooltip("Activate if you plan to call Purchases.Configure or Purchases.Setup programmatically.")]
```

Comment:
did a decent amount of cleanup on these tooltips so they look a bit better

Reply:
mostly line breaks both in C# by splitting the string and in the tooltip itself with `\n`